Week04/src/SalesRep.java

```diff
@@ -0,0 +1,50 @@
+public class SalesRep {
+
+    protected String name;
+    protected int sales;
+    protected int quota;
+    protected int revenue;
+
+    public SalesRep(String name, int sales, int quota, int revenue) {
+        this.name = name;
+        this.sales = sales;
+        this.quota = quota;
+        this.revenue = revenue;
```

Comment:
I would calculate revenue here an not have it as a parameter
revenue = sales * quota